src/Microsoft.Management.Infrastructure.CimCmdlets/GetCimAssociatedInstanceCommand.cs

```diff
@@ -90,7 +90,10 @@ public string ResultClassName
         [Alias(CimBaseCommand.AliasCimInstance)]
         public CimInstance InputObject
         {
-            get { return cimInstance; }
+            get
+            {
+                return cimInstance;
+            }
```

Comment:
It is auto fix? Is it possible to use modern `get => cimInstance;`?

Reply:
There is an open issue for expression-bodied accessors: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2402

Reply:
`get => cimInstance;` is not currently a violation, but may not be in the spirit of the rule.